README.md

```diff
@@ -211,34 +219,88 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
+
+1. **Input:** A valid payment file.
+**Output:** The transactions are logged, the payment file is moved from `inbound` to `archive`, and each valid payment request is sent to the Topic `PAYMENT_TOPIC_REQUEST`.
+
+2. **Input:** A payment file where the amount of payment requests does not match the batch header.
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.
+
+3. **Input:** A payment file with an invalid batch header (e.g. wrong date format)
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.
+
+4. **Input:** A payment file where the batch header value matches the payment line values but one or more payment line does not match the corresponding invoice line total.
+**Output:** The valid payment requests are logged and error messages are logged for each invalid request, the payment file is moved from `inbound` to `archive`, each valid payment request is sent to Topic `PAYMENT_TOPIC_ADDRESS`, and an invalid event is sent to `EVENT_TOPIC_ADDRESS` for each invalid payment request.
+
+5. **Input:** A payment file where the batch header sequence number is different from the expected value.
+**Output:** If the sequence number is lower than the expected value the payment file is ignored. If the sequence number is higher than the expected value the payment file is quarantined. A message is logged stating that the payment file has been ignored or quarentined accordingly.
+
+#### Example output message
```

Comment:
I would remove -- we're trying to steer away from harcoding messages into our READMEs -- instead we're looking to reference the messages in our `asyncapi` doc -- remind me to see where we can included these in the repos